packages/core/src/server/graphql/schema.test.ts

```diff
@@ -1,9 +1,7 @@
+import { createSchema } from "@/schema/schema.js";
```

Comment:
I would probably add this to other test file (`buildGraphqlSchema.test.ts`). Also, would consider renaming `buildGraphqlSchema.ts` to `schema.ts`, but leave it up to you.

Reply:
It's confusing to me that we have a lot of things already called schema